src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 726px;
```

Comment:
why 726?
use calculations for this value
checklist p6

Reply:
done